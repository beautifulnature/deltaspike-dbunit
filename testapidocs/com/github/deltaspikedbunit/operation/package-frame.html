<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_79) on Thu Nov 05 12:48:01 GMT 2015 -->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>com.github.deltaspikedbunit.operation (Deltaspike DBUnit 0.5.0 Test API)</title>
<meta name="date" content="2015-11-05">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/deltaspikedbunit/operation/package-summary.html" target="classFrame">com.github.deltaspikedbunit.operation</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="DefaultDatabaseOperationLookupTest.html" title="class in com.github.deltaspikedbunit.operation" target="classFrame">DefaultDatabaseOperationLookupTest</a></li>
<li><a href="MicrosoftSqlDatabaseOperationLookupTest.html" title="class in com.github.deltaspikedbunit.operation" target="classFrame">MicrosoftSqlDatabaseOperationLookupTest</a></li>
</ul>
</div>
</body>
</html>
