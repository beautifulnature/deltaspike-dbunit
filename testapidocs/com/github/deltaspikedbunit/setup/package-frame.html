<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_79) on Thu Oct 15 09:40:34 BST 2015 -->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>com.github.deltaspikedbunit.setup (Deltaspike DBUnit 0.5.0 Test API)</title>
<meta name="date" content="2015-10-15">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/deltaspikedbunit/setup/package-summary.html" target="classFrame">com.github.deltaspikedbunit.setup</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="CleanInsertSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">CleanInsertSetupOnClassTest</a></li>
<li><a href="CleanInsertSetupOnMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">CleanInsertSetupOnMethodTest</a></li>
<li><a href="DeleteAllSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">DeleteAllSetupOnClassTest</a></li>
<li><a href="DeleteAllSetupOnMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">DeleteAllSetupOnMethodTest</a></li>
<li><a href="DeleteSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">DeleteSetupOnClassTest</a></li>
<li><a href="DeleteSetupOnMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">DeleteSetupOnMethodTest</a></li>
<li><a href="InsertSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">InsertSetupOnClassTest</a></li>
<li><a href="InsertSetupOnMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">InsertSetupOnMethodTest</a></li>
<li><a href="MixedSetupOnClassAndMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">MixedSetupOnClassAndMethodTest</a></li>
<li><a href="MultipleInsertSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">MultipleInsertSetupOnClassTest</a></li>
<li><a href="RefreshSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">RefreshSetupOnClassTest</a></li>
<li><a href="RefreshSetupOnMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">RefreshSetupOnMethodTest</a></li>
<li><a href="TruncateSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">TruncateSetupOnClassTest</a></li>
<li><a href="TruncateSetupOnMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">TruncateSetupOnMethodTest</a></li>
<li><a href="UpdateSetupOnClassTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">UpdateSetupOnClassTest</a></li>
<li><a href="UpdateSetupOnMethodTest.html" title="class in com.github.deltaspikedbunit.setup" target="classFrame">UpdateSetupOnMethodTest</a></li>
</ul>
</div>
</body>
</html>
