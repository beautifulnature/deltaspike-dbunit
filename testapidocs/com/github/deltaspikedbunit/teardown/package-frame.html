<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_79) on Thu Oct 15 09:40:34 BST 2015 -->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>com.github.deltaspikedbunit.teardown (Deltaspike DBUnit 0.5.0 Test API)</title>
<meta name="date" content="2015-10-15">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/deltaspikedbunit/teardown/package-summary.html" target="classFrame">com.github.deltaspikedbunit.teardown</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="CleanInsertTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">CleanInsertTearDownOnClassTest</a></li>
<li><a href="CleanInsertTearDownOnMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">CleanInsertTearDownOnMethodTest</a></li>
<li><a href="DeleteAllTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">DeleteAllTearDownOnClassTest</a></li>
<li><a href="DeleteAllTearDownOnMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">DeleteAllTearDownOnMethodTest</a></li>
<li><a href="DeleteTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">DeleteTearDownOnClassTest</a></li>
<li><a href="DeleteTearDownOnMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">DeleteTearDownOnMethodTest</a></li>
<li><a href="InsertTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">InsertTearDownOnClassTest</a></li>
<li><a href="InsertTearDownOnMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">InsertTearDownOnMethodTest</a></li>
<li><a href="MixedTearDownOnClassAndMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">MixedTearDownOnClassAndMethodTest</a></li>
<li><a href="MultipleInsertTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">MultipleInsertTearDownOnClassTest</a></li>
<li><a href="RefreshTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">RefreshTearDownOnClassTest</a></li>
<li><a href="RefreshTearDownOnMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">RefreshTearDownOnMethodTest</a></li>
<li><a href="TruncateTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">TruncateTearDownOnClassTest</a></li>
<li><a href="TruncateTearDownOnMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">TruncateTearDownOnMethodTest</a></li>
<li><a href="UpdateTearDownOnClassTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">UpdateTearDownOnClassTest</a></li>
<li><a href="UpdateTearDownOnMethodTest.html" title="class in com.github.deltaspikedbunit.teardown" target="classFrame">UpdateTearDownOnMethodTest</a></li>
</ul>
</div>
</body>
</html>
