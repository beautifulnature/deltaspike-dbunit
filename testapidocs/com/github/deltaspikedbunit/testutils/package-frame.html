<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_79) on Thu Oct 15 10:16:26 BST 2015 -->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>com.github.deltaspikedbunit.testutils (Deltaspike DBUnit 0.5.0 Test API)</title>
<meta name="date" content="2015-10-15">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/deltaspikedbunit/testutils/package-summary.html" target="classFrame">com.github.deltaspikedbunit.testutils</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="ExpectedFailureInterceptor.html" title="class in com.github.deltaspikedbunit.testutils" target="classFrame">ExpectedFailureInterceptor</a></li>
</ul>
<h2 title="Annotation Types">Annotation Types</h2>
<ul title="Annotation Types">
<li><a href="ExpectedFailure.html" title="annotation in com.github.deltaspikedbunit.testutils" target="classFrame">ExpectedFailure</a></li>
</ul>
</div>
</body>
</html>
