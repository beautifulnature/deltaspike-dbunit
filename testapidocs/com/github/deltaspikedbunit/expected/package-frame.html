<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_79) on Thu Oct 15 10:16:26 BST 2015 -->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>com.github.deltaspikedbunit.expected (Deltaspike DBUnit 0.5.0 Test API)</title>
<meta name="date" content="2015-10-15">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<h1 class="bar"><a href="../../../../com/github/deltaspikedbunit/expected/package-summary.html" target="classFrame">com.github.deltaspikedbunit.expected</a></h1>
<div class="indexContainer">
<h2 title="Classes">Classes</h2>
<ul title="Classes">
<li><a href="ExpectedFailureOnClassTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedFailureOnClassTest</a></li>
<li><a href="ExpectedFailureOnMethodTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedFailureOnMethodTest</a></li>
<li><a href="ExpectedNonStrictOnClassTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedNonStrictOnClassTest</a></li>
<li><a href="ExpectedNonStrictOnMethodTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedNonStrictOnMethodTest</a></li>
<li><a href="ExpectedNonStrictUnorderedOnClassTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedNonStrictUnorderedOnClassTest</a></li>
<li><a href="ExpectedNonStrictUnorderedOnMethodTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedNonStrictUnorderedOnMethodTest</a></li>
<li><a href="ExpectedOnClassAndMethodTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedOnClassAndMethodTest</a></li>
<li><a href="ExpectedOnClassAndMethodWithoutOverrideTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedOnClassAndMethodWithoutOverrideTest</a></li>
<li><a href="ExpectedQueryNonStrictOnMethodTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedQueryNonStrictOnMethodTest</a></li>
<li><a href="ExpectedQueryOnMethodTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedQueryOnMethodTest</a></li>
<li><a href="ExpectedQueryWithModifierOnMethodTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedQueryWithModifierOnMethodTest</a></li>
<li><a href="ExpectedTableNonStrictOnClassTest.html" title="class in com.github.deltaspikedbunit.expected" target="classFrame">ExpectedTableNonStrictOnClassTest</a></li>
</ul>
</div>
</body>
</html>
